Monika After Story/game/cardgames.rpy

```diff
@@ -0,0 +1,4781 @@
+                del[menu_items]
+                del[final_item]
+            return
+
+        elif _return == "win_points":
+            m 1eub "Alright!"
+            call monika_nou_change_house_rules.change_win_points_loop
+
+        elif _return == "start_cards":
+            m 1eub "Alright!"
+            call monika_nou_change_house_rules.change_starting_cards_loop
+
+        elif _return == "stack_d2":
+            if not persistent._mas_game_nou_house_rules["stack_d2"]:
+                m 1tub "Okay, but I must warn you that that might go against you~"
+
+            else:
+                m 1ttu "Did I go too hard at this game for you?"
+                m 1hub "Ahaha~ I'm just kidding!"
+
+            $ persistent._mas_game_nou_house_rules["stack_d2"] = not persistent._mas_game_nou_house_rules["stack_d2"]
+
+        elif _return == "play_wd4_anytime":
+            if not persistent._mas_game_nou_house_rules["play_wd4_anytime"]:
+                # m "Oh, you better be ready for this one, [player]~"
+                m 1eua "That sounds fun."
+
+            else:
+                m 1eua "Back to the classic rules, I see."
+
+            $ persistent._mas_game_nou_house_rules["play_wd4_anytime"] = not persistent._mas_game_nou_house_rules["play_wd4_anytime"]
+
+        else:
+            m 3eub "Okay! Then settled!"
+
+            python:
+                persistent._mas_game_nou_house_rules["win_points"] = 200
+                persistent._mas_game_nou_house_rules["start_cards"] = 7
+                persistent._mas_game_nou_house_rules["stack_d2"] = False
+                persistent._mas_game_nou_house_rules["play_wd4_anytime"] = False
+
+                persistent._mas_game_nou_points["Monika"] = 0
+                persistent._mas_game_nou_points["Player"] = 0
+
+                del[is_default_rules]
+                del[menu_items]
+                del[final_item]
+
+            return
+
+    $ persistent._mas_game_nou_points["Monika"] = 0
+    $ persistent._mas_game_nou_points["Player"] = 0
+
+    m 3eua "Is there anything else you would like to change?{nw}"
+    $ _history_list.pop()
+    menu:
+        m "Is there anything else you would like to change?{fast}"
+
+        "Yes.":
+            jump monika_nou_change_house_rules.selection_loop
+
+        "No.":
+            m 2eub "Then let's play together soon~"
+
+    python:
+        del[is_default_rules]
+        del[menu_items]
+        del[final_item]
+
+    return
+
+label .change_win_points_loop:
+    $ ready = False
+    while not ready:
+        show monika 1eua at t11
+
+        $ points_cap = store.mas_utils.tryparseint(
+            renpy.input(
+                "How many points would you like it to be?",
+                allow=numbers_only,
+                length=4
+            ).strip("\t\n\r"),
+            200
+        )
+
+        if points_cap < 0:
+            m 2rksdla "[player], how would we play with negative amount of point?"
```

Comment:
Grammar is wrong here. If it is impossible to lose points (I assume), then might I recommend "[player], the game will never end if the goal is negative."